.github/workflows/intellijPluginUpload.yml

```diff
@@ -0,0 +1,27 @@
+name: Build and Publish Intellij Plugin
+on:
+  release:
+    types: [published]
+jobs:
+  trigger_workflow:
+     strategy:
+       matrix:
+         os: [ubuntu-latest]
+     runs-on: ${{ matrix.os }}
+     env:
+         working-directory: ./tool-plugins/intellij
+         publish_token: ${{ secrets.PUBLISH_TOKEN }}
```

Comment:
Shall we rename this as something like "IDEA_TOKEN"?

Reply:
Done